docs/streams/upgrade-guide.html

```diff
@@ -34,9 +34,17 @@ <h1>Upgrade Guide and API Changes</h1>
     </div>
 
     <p>
-        If you are using Java 7 and want to upgrade from 1.0.x to 1.1.0 you don't need to make any code changes as the public API is fully backward compatible.
-        If you are using Java 8 method references in your Kafka Streams code you might need to update your code to resolve method ambiguties.
-	Hot-swaping the jar-file only might not work for this case.
+        If you want to upgrade from 1.1.x to 1.2.0 and you have customized window store implementations on the <code>ReadOnlyWindowStore</code> interface
```

Comment:
@guozhangwang @dguy @rajinisivaram This PR was cherry-picked to 1.1 branch -- however, this paragraph talks about 1.2 release and should not be part of 1.1. I don't think we need to get a new RC for this, but we should make sure to update the docs before we push them live for 1.1. Thanks.

Reply:
Thanks for pointing it out!

Reply:
@mjsax @guozhangwang Is there a PR to fix this in the 1.1 branch? If not, can you submit one so that I can generate 1.1 docs using that? Or do I just need to revert this commit to generate the docs? Thanks.